closed/Intel/code/offline_test.sh

```diff
@@ -1,15 +1,28 @@
-cd 3d-unet-99.9/pytorch-cpu-kits19/
-../../run_docker.sh ce03546c4d75  /workspace/run_offline.sh /opt/workdir/code/3d-unet-99.9/pytorch-cpu-kits19 aws_3dunet  3dunet.txt
+cd resnet50/pytorch-cpu/
+chmod +x run_offline_accuracy.sh
+chmod +x run_offline.sh
+../../run_docker.sh a17e4ea194cc  /workspace/run_offline_accuracy.sh  /opt/workdir/code/resnet50/pytorch-cpu aws_rn50  resnet50_offline_acc.txt
```

Comment:
might need to replace instance numbers with intel/intel-optimized-pytorch:mlperf-inference-2.1-3dunet instead. The number will be different among users' env.